server/svix_test_suite/src/lib.rs

```diff
@@ -0,0 +1,93 @@
+// Everything is unused because only `cargo test` is used with this crate
+#![allow(dead_code)]
+
+/// A crate meant for end-to-end integration testing of the Svix server. These tests assume an
+/// already running Svix server and authentication token which are given to the tests via the
+/// environmental variables `SVIX_SERVER_URI` and `SVIX_SERVER_TOKEN`.
+mod request_util;
+#[cfg(test)]
+mod tests;
+
+use std::{
+    env,
+    net::{SocketAddr, TcpListener},
+};
+
+use async_trait::async_trait;
+use axum::{
+    extract::{FromRequest, RequestParts},
+    headers::HeaderMap,
+    routing::post,
+    Json, Router,
+};
+use tokio::sync::{mpsc, oneshot};
+
+pub struct SvixUri(pub String);
+pub struct SvixToken(pub String);
+
+/// Fetches the environmental variables needed to connect to and authenticate with a running Svix
+/// server
+fn fetch_env_vars() -> (SvixUri, SvixToken) {
+    let _ = dotenv::dotenv();
+
+    (
+        SvixUri(env::var("SVIX_SERVER_URL").unwrap()),
+        SvixToken(env::var("SVIX_AUTH_TOKEN").unwrap()),
```

Comment:
This you don't want to fetch, but rather generate one with a custom org_id on every test.